gossip/src/crds_entry.rs

```diff
@@ -57,23 +57,17 @@ impl_crds_entry!(LegacyContactInfo, CrdsData::LegacyContactInfo(node), node);
 impl_crds_entry!(LegacyVersion, CrdsData::LegacyVersion(version), version);
 impl_crds_entry!(LowestSlot, CrdsData::LowestSlot(_, slot), slot);
 impl_crds_entry!(Version, CrdsData::Version(version), version);
+impl_crds_entry!(
+    SnapshotHashes,
+    CrdsData::SnapshotHashes(snapshot_hashes),
+    snapshot_hashes
+);
 impl_crds_entry!(
     IncrementalSnapshotHashes,
     CrdsData::IncrementalSnapshotHashes(incremental_snapshot_hashes),
     incremental_snapshot_hashes
 );
 
-impl<'a, 'b> CrdsEntry<'a, 'b> for &'a SnapshotHashes {
```

Comment:
This manual implementation was because up until https://github.com/solana-labs/solana/pull/20421, this was called `SnapshotHash` vs `SnapshotHashes`; so the existing macro wouldn't work here.